src/ontology/uberon.Makefile

```diff
@@ -1168,95 +1184,114 @@ composites: composite-metazoan.obo composite-vertebrate.obo
 	if [ -f $< ]; then if ! cmp $< $@ ; then cat $< > $@ ; fi ; fi
 
-# Generate cross-references from the FBbt mapping file
-$(COMPONENTSDIR)/mappings.owl: mappings/fbbt-mappings.sssom.tsv ../scripts/sssom2xrefs.awk
-	awk -f ../scripts/sssom2xrefs.awk $< > $@
 
-
-# Prepare Uberon and CL sources
 # ----------------------------------------
-# Uberon: We need to forcefully merge the mappings component generated
-# from the FBbt mappings so that the FBbt cross-references will be
-# visible to the bridge-generating script.
-$(TMPDIR)/seed.obo: $(SRC) $(COMPONENTSDIR)/mappings.owl
-	$(ROBOT) merge -i $< -i $(COMPONENTSDIR)/mappings.owl --collapse-import-closure false \
-		 convert -f obo --check false -o $@.tmp && \
-		$(SCRIPTSDIR)/obo-grep.pl --neg -r is_obsolete $@.tmp > $@
-
-# CL (step 1): We take the full ontology, excluding external classes,
-# but including references to those
-$(TMPDIR)/cl-core.obo: $(SRC)
-	$(OWLTOOLS) $(URIBASE)/cl.owl \
-		--make-subset-by-properties -n BFO:0000050 BFO:0000051 RO:0002202 RO:0002215 \
-		--remove-external-classes -k CL \
-		--remove-dangling --remove-axiom-annotations --remove-imports-declarations \
-		-o -f obo --no-check $@
-
-# CL: (step 2): We add the header tags that the bridge-generating script
-# is relying upon
-$(TMPDIR)/cl-with-xrefs.obo: $(TMPDIR)/cl-core.obo $(SCRIPTSDIR)/expand-idspaces.pl
-	if [ $(BRI) = true ]; then \
-		egrep '^(idspace|treat-)' $(SRC) > $@.tmp && \
-		cat $< >> $@.tmp && \
-		$(SCRIPTSDIR)/expand-idspaces.pl $@.tmp > $@; \
-	fi
-
-# ZFA requires special treatment because most xrefs between CL and ZFA
-# are on the ZFA side, so they need to be extracted and inverted before
-# we can generate the ZFA bridges out of them.
-# FIXME: This is currently broken: the step below does not extract
-# anything, which is probably why the CL-to-ZFA bridge is almost empty
-# (https://github.com/obophenotype/uberon/issues/1813)
-$(TMPDIR)/cl-zfa-xrefs.obo: mirror/zfa.owl
-	if [ $(MIR) = true ] && [ $(IMP) = true ]; then \
-		$(ROBOT) query -i $< --query ../sparql/zfa-xrefs-to-cl.sparql $@_xrefs_to_zfa.tsv && \
-		cat $@_xrefs_to_zfa.tsv | tail -n +2 | \
-		$(SCRIPTSDIR)/tbl2obolinks.pl --rel xref - > $@.tmp && mv $@.tmp $@; \
-	fi
-
-
-# Building the bridges
+# BRIDGES
 # ----------------------------------------
-# Most bridges are built here, by a script that extracts xrefs from
-# the source files prepared above and derives the bridges from them.
-$(TMPDIR)/bridges: $(TMPDIR)/seed.obo $(TMPDIR)/cl-with-xrefs.obo $(TMPDIR)/cl-zfa-xrefs.obo $(REPORTDIR)/life-cycle-xrefs.txt $(CUSTOM_BRIDGES)
-	if [ $(BRI) = true ]; then \
-		cd $(BRIDGEDIR) && \
-		perl ../../scripts/make-bridge-ontologies-from-xrefs.pl -l ../$(REPORTDIR)/life-cycle-xrefs.txt ../$(TMPDIR)/seed.obo && \
-		perl ../../scripts/make-bridge-ontologies-from-xrefs.pl -l ../$(REPORTDIR)/life-cycle-xrefs.txt -b cl ../$(TMPDIR)/cl-with-xrefs.obo ../$(TMPDIR)/cl-zfa-xrefs.obo && \
-		cd .. && touch $@; \
-	fi
+ifeq ($(BRI),true)
```

Comment:
Wow what a cool pattern, cc @anitacaron:

1. Evaluate the condition on makefile (not shell!) level: `ifeq ($(BRI),true)`
2. add all dependencies needed in the IF clock below
3. don't include anything in the else part

This is a great way to _really_ skip certain areas of the code you don't want to run (could be used for the DOSDP), for example.

Reply:
For reference, this is the idea mentioned in this ticket on the ODK: https://github.com/INCATools/ontology-development-kit/issues/912